website/source/docs/drivers/docker.html.md

```diff
@@ -728,6 +728,10 @@ plugin "docker" {
       `docker.volumes.enabled` set to false, the labels will still be applied to
       the standard binds in the container.
 
+* `infra_image` - This is the docker image to use when creating the parent
```

Comment:
Is `infra` the right term for this image? Isn't it more commonly referred to as a `pause` image? Or we could name it after its use: `net_image`. Not a blocker. The name just seems overly generic for the very specific use it seems to serve.

```suggestion
* `infra_image` - This is the Docker image to use when creating the parent
```